bib/internal/setup/setup_test.go

```diff
@@ -61,3 +61,13 @@ func TestValidateCanRunTargetArchUnexpectedOutput(t *testing.T) {
 	err := setup.ValidateCanRunTargetArch("fakearch")
 	assert.ErrorContains(t, err, `internal error: unexpected output`)
 }
+
+func TestValidateTags(t *testing.T) {
```

Comment:
At this level I would prefer if we would not call out to the network. We could generate a local container with the right locals via `podman build` or we just fake the podman binary in the test and also add an intergration test to the integration test suite. I lean towards the later (as the integration test is needed anyway and it is a bit easier with the existing code) - but I have no super strong opinion here. I attached to git commits that I can push here (if you want) or that you could `git am` into your tree with what I have in mind for the testing of this feature:

[0001-setup-tweak-TestValidateTags-to-fake-the-podman-bina.patch.txt](https://github.com/user-attachments/files/15882957/0001-setup-tweak-TestValidateTags-to-fake-the-podman-bina.patch.txt)
[0002-test-add-new-integration-test-for-ValidateHasContain.patch.txt](https://github.com/user-attachments/files/15882958/0002-test-add-new-integration-test-for-ValidateHasContain.patch.txt)

Reply:
I don't like having a non-automated test, so I'm for whatever gets automation. I'll look at and apply those patches.